src/lib/Components/OpaqueImageView.tsx

```diff
@@ -25,6 +25,8 @@ interface Props {
   /** Any additional styling for the imageview */
   style?: any
 
+  skipGemini?: boolean
```

Comment:
lol, yea so the `OpaqueImageView` seems to have an overlay and then a resized on-the-fly image.

However...in testing yesterday it looks like the S3 URL's that are returned by Radiation which links to the attachment, error out when trying to resize with Gemini.

So this means that for now, it just renders the actual attachment (even for the tiny preview).

Later today I'll discuss with @joeyAghion / @ashkan18 the possibility of fixing up how Gemini resizes these (to avoid the error), or (possibly better), pre-render small thumbnails for image attachments. Then we could just serve those directly.